src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -122,6 +122,33 @@ const PIAFormPage = () => {
     }));
   };
 
+  /**
+   * Reset pia.review when pia.status is changed to INCOMPLETE or EDIT_IN_PROGRESS
+   */
```

Comment:
we can remove these as backend will handle this, you are just handle prohibit user change url right